java.d/src/main/java/org/firehol/netdata/module/jmx/store/MBeanLongStore.java

```diff
@@ -0,0 +1,15 @@
+package org.firehol.netdata.module.jmx.store;
+
+import org.firehol.netdata.model.Dimension;
+
+public class MBeanLongStore extends MBeanValueStore {
+
+	protected void prepareDimension(Dimension dimension) {
+		dimension.setDivisor(dimension.getDivisor());
```

Comment:
This does nothing. I think we can use the default implementation of `prepareDimension`.